src/components/todo.js

```diff
@@ -1,27 +1,35 @@
 import { getItems, setItems } from '../utilities/localStorage';
 
-const TodoElement = (content, isActive, id) => {
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you fix your `Edit` task description feature even if it was requirement of your previous milestone, as **right now it is no longer working as expected, changes are not save in local storage after editing**. This will make your application still keeping all the expected functionalities.